recipes/biscuit/meta.yaml

```diff
@@ -21,20 +21,19 @@ requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - zlib
-    - ncurses
-    - libcurl
+    - zlib >=1.2.11
```

Comment:
This should not be needed? Can you describe why you add this lower-pins?

Reply:
Thank you for reviewing!

And yes, I need it to work with `samtools 1.9`, and that's what helped me when I tried locally (with `conda --use-local`)

Reply:
@bgruening for context see: https://github.com/nf-core/methylseq/pull/186#discussion_r580174146 (we're trying to get this package to work in a fairly large environment that needs to run Samtools v1.9).

Reply:
@bgruening Any news with this?

Reply:
Sorry, not good times here.

I don't think this is the correct solution. Those libraries are pinned globally. This means we set for all compiled packages a zlib version. And if we upgrade the zlib version we recompile all packages (assuming ABI instability etc).

So it should never be needed to pin zlib to a different version and could have a bad impact on your environment.

Can you descripe maybe the underlying problem that you want to solve. I hope we find a better solution :)

Reply:
Yes, thanks.

When using `samtools 1.9` in the environment.yml file, the biscuit 0.3.16 default build number is 0. 
When using `samtools 1.10` in the environment.yml file, the biscuit default build is 3.

So I need that the default build for `samtools 1.9` would be biscuit 0.3.16 build 3.

Reply:
why can you not specify everything in your environment.yml file, including the build number?

Reply:
Because then I get a conflict:

The environment.yml file:
```

name: nf-core-methylseq-1.5
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - bioconda::htslib=1.9
  - bioconda::samtools=1.9
  - bioconda::biscuit=0.3.16.20200420=h2b0c03c_3
```

Output for `conda env create -f environment.yml`:
```
Collecting package metadata (repodata.json): done
Solving environment: -
Found conflicts! Looking for incompatible packages.                                                                                     failed
Solving environment: /
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.                                                                                  failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package htslib conflicts for:
bioconda::htslib=1.9
bioconda|bioconda::htslib=1.9
bioconda::samtools=1.9 -> htslib[version='>=1.9,<1.10.0a0']

Package _libgcc_mutex conflicts for:
bioconda::biscuit==0.3.16.20200420=h2b0c03c_3 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main']
bioconda::htslib=1.9 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main']
bioconda::samtools=1.9 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main']

Package ncurses conflicts for:
bioconda::biscuit==0.3.16.20200420=h2b0c03c_3 -> ncurses[version='>=6.2,<6.3.0a0']
bioconda::samtools=1.9 -> ncurses[version='>=5.9,<5.10.0a0|>=6.1,<6.2.0a0']

Package libgcc-ng conflicts for:
bioconda::biscuit==0.3.16.20200420=h2b0c03c_3 -> libgcc-ng[version='>=7.5.0']
bioconda::biscuit==0.3.16.20200420=h2b0c03c_3 -> libcurl[version='>=7.71.1,<8.0a0'] -> libgcc-ng[version='>=4.9|>=7.3.0|>=9.3.0|>=7.2.0']
```

If I use the same environment.yml file, but without specifying the build number, the environment is created OK, but when running `conda list` inside the environment, I get:
```
# packages in environment at /path/miniconda3/envs/check_biscuit:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
biscuit                   0.3.16.20200420      h802ef42_1    bioconda
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.17.1               h7f98852_1    conda-forge
ca-certificates           2020.12.5            ha878542_0    conda-forge
curl                      7.75.0               h979ede3_0    conda-forge
htslib                    1.9                  h244ad75_9    bioconda
krb5                      1.17.2               h926e7f8_0    conda-forge
libcurl                   7.75.0               hc4aaa36_0    conda-forge
libdeflate                1.3                  h516909a_0    conda-forge
libedit                   3.1.20191231         h46ee950_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libgcc-ng                 9.3.0               h2828fa1_18    conda-forge
libgomp                   9.3.0               h2828fa1_18    conda-forge
libnghttp2                1.43.0               h812cca2_0    conda-forge
libssh2                   1.9.0                hab1572f_5    conda-forge
libstdcxx-ng              9.3.0               h6de172a_18    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openssl                   1.1.1j               h7f98852_0    conda-forge
samtools                  1.9                 h10a08f8_12    bioconda
tk                        8.6.10               h21135ba_1    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
zlib                      1.2.11            h516909a_1010    conda-forge

```

That means that the default biscuit build is 1, and I want it to be 3

Reply:
@bgruening ...

Reply:
@bgruening any news with this..?